### ISSUE TITLE
react-axe shows a critical issue for the TabSlider

### ISSUE THREAD

**Issue by mtblanton** [palmetto/palmetto-components#478]
react-axe is an a11y checker that checks the rendered dom for us.

The error:
critical: Certain ARIA roles must contain particular children https://dequeuniversity.com/rules/axe/3.5/aria-required-children?application=axeAPI index.js:216

Believe this is from the `div` inside the `ul` in that component, but not 100% sure!

```
<ul class="flex-direction-row display-flex font-size-inherit position-relative" style="padding-inline-start:0"
  role="tablist">
  <li class="tab-item__2koth tab-item--selected__Vz4x8 p-0 flex-direction-column display-flex font-size-sm"
    style="flex:1;cursor:pointer" role="presentation"><button
      class="p-v-xs p-h-0 flex-direction-row display-flex justify-content-center align-items-center font-size-inherit border-width-0 font-color-grey-400"
      style="background:transparent;white-space:nowrap">Loan</button></li>
  <li class="tab-item__2koth p-0 flex-direction-column display-flex font-size-sm" style="flex:1;cursor:pointer"
    role="presentation"><button
      class="p-v-xs p-h-0 flex-direction-row display-flex justify-content-center align-items-center font-size-inherit border-width-0 font-color-grey-400"
      style="background:transparent;white-space:nowrap">Cash</button></li>
  <div
    class="tabs-slider-indicator__20scv br-md h-100 flex-direction-column display-flex font-size-inherit border-width-sm position-absolute background-color-white border-color-grey-100"
    style="left: 0px; width: 63.8px;"></div>
</ul>
```

**Comment by juanfabrega** [comment]
@mtblanton awesome catch, thanks! Will have an updated package this week for this.

**Comment by nathanyoung** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Minor bugs](https://trello.com/c/6KAmKTHK/685-minor-bugs)

**Comment by juanfabrega** [comment]
@mtblanton this is addressed here: https://github.com/palmetto/palmetto-components/pull/504

available in `0.74.6`
